app/views/volunteers/index.html.slim

```diff
@@ -1,25 +1,24 @@
 .row
   .col-xs-12
     h1= t('volunteers')
-
+.row
+  .col-xs-12= render 'state_select', size: 'md', descriptive: true
 .row
   .col-xs-12
   table.table.table-striped
     thead
       tr
-        th= t_attr(:first_name)
-        th= t_attr(:last_name)
-        th= t_attr(:date_of_birth)
-        th= t_attr(:nationality)
-        th= t_attr(:gender)
-        th= t_attr(:email)
-        th
-        th
-        th
+        th= sort_link @q, :first_name, t_attr(:full_name)
+        th= sort_link @q, :date_of_birth
+        th= sort_link @q, :nationality
+        th= sort_link @q, :created_at
+        th= sort_link @q, :gender
+        th= render 'state_select'
```

Comment:
I whould prefer, the table headers to be only used for sorting, same like the other ths